examples/mxnet/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM horovod/horovod:0.16.2-tf1.12.0-torch1.1.0-mxnet1.4.1-py3.5 AS build
+
+# Create a wrapper for OpenMPI to allow running as root by default
```

Comment:
These lines seem to have been removed in horovod's Dockerfile in favor of `horovodrun`. Have you tried to use it?

Reply:
I haven't tried horovodrun. Should this example be using horovodrun?

Reply:
I guess it's up to you. We can certainly do it at a later time.